doc/tutorials/using_partition_mode_on_up2.rst

```diff
@@ -0,0 +1,136 @@
+.. _partition_mode:
+
+Using partition mode on UP2
+===========================
+This tutorial describes all the steps you need to take in order to use partition mode with the ACRN hypervisor on UP2.
+
+Enabling partition mode
+***********************
+1. Make the following changes in ``hypervisor/arch/x86/Kconfig`` :
```

Comment:
This really should be done using `make menuconfig`, i.e. something like:
```
make defconfig PLATFORM=sbl
make menuconfig
```
and change both options